university-of-aleppo-faculty-of-medicine.csl

```diff
@@ -0,0 +1,97 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style class="in-text" version="1.0" xmlns="http://purl.org/net/xbiblio/csl">
+  <info>
+    <title>University of Aleppo - Faculty of Medicine</title>
+    <title-short>UAFM</title-short>
+    <id>http://www.zotero.org/styles/university-of-aleppo-faculty-of-medicine</id>
+    <link rel="self" href="http://www.zotero.org/styles/university-of-aleppo-faculty-of-medicine"/>
+    <author>
+      <name>Firas Khalil Khana</name>
+      <email>firasuke@gmail.com</email>
+    </author>
+    <category citation-format="numeric"/>
+    <category field="medicine"/>
+    <issn>1022-680X</issn>
+    <summary>Created for use in articles, dissertations and theses in the Faculty of Medicine at the University of Aleppo</summary>
+    <updated>2019-05-04T14:04:33+00:00</updated>
+    <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
+  </info>
+  <macro name="author">
+    <names variable="author">
+      <name initialize-with=". " name-as-sort-order="all" sort-separator=" "/>
+    </names>
+  </macro>
+  <macro name="title">
+    <text variable="title" text-case="title"/>
+  </macro>
+  <macro name="date">
+    <date date-parts="year" form="text" variable="issued"/>
+  </macro>
+  <macro name="publisher">
+    <text variable="publisher"/>
+  </macro>
+  <macro name="edition">
+    <number vertical-align="baseline" suffix=" ed. " variable="edition" form="ordinal"/>
```

Comment:
no need for the vertical-align. Better to use a term instead of a suffix for edition. Also, good to account for non-number editions (e.g. "completely revised 2nd edition")